test/integration/kprobe/base/bytecode.yaml

```diff
@@ -0,0 +1,18 @@
+---
```

Comment:
Hrm so I don't think we should be duplicating all these here, let's talk more about the best way to install the example manifests when they live in their own repo, I'm thinking we could just maintain a static yaml and do something like 

`kustomize apply -k https://raw.githubusercontent.com/bpfman/bpfman/main/examples/config/base/go-xdp-counter/<FULL YAML>`